cirq-core/cirq/transformers/drop_negligible_operations_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cirq
+
+NO_COMPILE_TAG = "no_compile_tag"
+
+
+def test_leaves_big():
+    a = cirq.NamedQubit('a')
+    circuit = cirq.Circuit([cirq.Moment([cirq.Z(a) ** 0.1])])
```

Comment:
nit: can remove lists
```suggestion
    circuit = cirq.Circuit(cirq.Moment(cirq.Z(a) ** 0.1))
```
Same thing applies throughout this module.